stubs/waitress/waitress/utilities.pyi

```diff
@@ -43,7 +43,7 @@ class Error:
     body: str = ...
     def __init__(self, body: str) -> None: ...
     def to_response(self) -> tuple[str, Sequence[tuple[str, str]], str]: ...
-    def wsgi_response(self, environ: Any, start_response: Callable[[str, Sequence[tuple[str, str]]], None]) -> str: ...
+    def wsgi_response(self, environ: Any, start_response: Callable[[str, Sequence[tuple[str, str]]], object]) -> str: ...
```

Comment:
Should probably use `_typeshed.wsgi.StartResponse`. For example, the second argument is a list in the implementation, which is actually required by the WSGI protocol. On the other hand, the return type is ignored, but I still think it's safer to use the predefined type.